pkg/build/pipelines/go/bump.yaml

```diff
@@ -0,0 +1,46 @@
+name: Bump go deps to a certain version
+
+needs:
+  packages:
+    - git
+
+inputs:
+  deps:
+    description: The deps to bump, space separated
+    required: true
+  modroot:
+    description: The root of the module
+    default: .
+  go-version:
+    description: "The go version to set the go.mod syntax to"
+    default: ""
+  replaces:
+    description: "The replaces to add to the go.mod file"
+  tidy:
+    default: true
+    description: Run go mod tidy command before and after the bump
+  show-diff:
+    default: false
+    description: Show the difference between the go.mod file before and after the bump
+  tidy-compat:
+    description: "Set the go version for which the tidied go.mod and go.sum files should be compatible"
+    default: ""
+
+pipeline:
+  - runs: |
+      # install the latest version of gobump
+      go install github.com/chainguard-dev/gobump@latest
```

Comment:
Given build systems are expected to be rigid, we do want to use a packaged version of gobump - rather than building new one on every go package; and/or arbitrary versioned one.

Imho this should still be the same as the stock wolfi-dev/os pipeline and add gobump in the environment.

Reply:
Fixed this, but will need re-review by someone else.